lib/node_modules/@stdlib/string/len/lib/next_break.js

```diff
@@ -0,0 +1,98 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2020 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var codePointAt = require( './code_point_at.js' );
+var isSurrogate = require( './is_surrogate.js' );
+var shouldBreak = require( './should_break.js' );
+var getGraphemeBreakProperty = require( './get_grapheme_break_property.js' );
+var getEmojiProperty = require( './get_emoji_property.js' );
+
+
+// MAIN //
+
+/**
+* Returns the next grapheme break in the string after the given index.
+*
+* @param {string} str - input string
+* @param {integer} idx - position in string for starting search
+* @returns {integer} next grapheme break position
+*
+* @example
+* var out = nextBreak( 'last man standing', 4 );
+* // returns 5
+*
+* @example
+* var out = nextBreak( 'presidential election', 8 );
+* // returns 9
+*
+* @example
+* var out = nextBreak( 'अनुच्छेद', 1 );
+* // returns 3
+*
+* @example
+* var out = nextBreak( '🌷', 0 );
+* // returns 2
+*/
+function nextBreak( str, idx ) {
+	var jsStrLength;
+	var prevEmoji;
+	var nextEmoji;
+	var midEmoji;
+	var prevCP;
+	var nextCP;
+	var prev;
+	var next;
+	var mid;
+	var i;
+
+	jsStrLength = str.length;
```

Comment:
```suggestion
	len = str.length;
```

I don't think a more verbose variable name is necessary here.